tmt/steps/report/reportportal.py

```diff
@@ -393,7 +394,14 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
             suite_description += f"<br>{self.data.artifacts_url}"
 
         # Communication with RP instance
-        with tmt.utils.retry_session() as session:
+        with tmt.utils.retry_session(status_forcelist=(
+                    429,   # Too Many Requests
+                    500,   # Internal Server Error
+                    502,   # Bad Gateway
+                    503,   # Service Unavailable
+                    504,   # Gateway Timeout
+                    5000,  # Unclassified error - weird error from RP appearing sometimes
```

Comment:
`5000` is not an http error?

Reply:
right, I thought it gets propagated to the log message but it is not. I will try to reproduce the issue to figure it out.

Reply:
So that was a 500 error in the end.